seisidd/devices/motors.py

```diff
@@ -89,25 +89,25 @@ def resume_position(self):
         self.rot.mv(self.position_cached['rot'])
 
         self.x_base.mv(self.position_cached['x_base'])
-        self.y_base.mv(self.position_cached['y_base'])
-        self.z_base.mv(self.position_cached['z_base'])
+        self.tiltx_base.mv(self.position_cached['tiltx_base'])
+        self.tiltx_base.mv(self.position_cached['tiltz_base'])
 
 
 class SimStageAero(MotorBundle):
     """
     Simulated Motor stacks used for HT-HEDM
     i.e.:   6iddSIM:m1
     Sim motors assigned as follows:
-        kx      = m1
-        ky      = m2
-        kz      = m3
-        rot     = m4
+        kx          = m1
+        ky          = m2
+        kz          = m3
+        rot         = m4
 
-        kx_tilt = m16
-        kz_tilt = m16
-        x_base  = m16
-        y_base  = m16
-        z_base  = m16
+        kx_tilt     = m16
+        kz_tilt     = m16
+        x_base      = m16
+        tiltx_base  = m16
+        tiltz_base  = m16
```

Comment:
We have 16 channels for the simulated motors, so we could give then different ones if necessary.